packages/nocodb/src/schema/swagger.json

```diff
@@ -20855,6 +20855,28 @@
           "id": "p1g7xrgdsn540"
         }
       },
+      "StringOrNullOrBooleanOrNumber": {
+        "description": "Model for StringOrNullOrBooleanOrNumber",
+        "examples": [
+          "string"
+        ],
+        "oneOf": [
+          {
+            "maxLength": 255,
```

Comment:
If I use `text` type, then it's possible to exceed 255 characters.

Reply:
Will remove the maxLength